pac/using-repository-crd.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="using-repository-crd"]
+include::_attributes/common-attributes.adoc[]
+= Using the Repository custom resource
+:context: using-repository-crd
+
+toc::[]
+[role="_abstract"]
+The `Repository` custom resource (CR) has the following primary functions:
```

Comment:
@mramendi Why is everything appearing in italics?

Reply:
GitHub issue, not present in preview build